Gems/ROS2/Code/Source/Camera/CameraSensor.cpp

```diff
@@ -128,6 +130,39 @@ namespace ROS2
         m_view->SetViewToClipMatrix(m_cameraSensorDescription.m_viewToClipMatrix);
         m_scene = AZ::RPI::RPISystemInterface::Get()->GetSceneByName(AZ::Name("Main"));
 
+        // In console mode, since there is no default pipeline, a BRDF texture needs to be manually initialized.
+        AZ::ApplicationTypeQuery appType;
+        AZ::ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationBus::Events::QueryApplicationType, appType);
```

Comment:
Could the brdf pipeline simply not be added within BootstrapSystemComponent::EnsureDefaultRenderPipelineInstalledForScene instead of adding it here?

Reply:
I would say "that depends". 

I think that, as a rule, console-mode shouldn't come with a default pipeline, because the rendering is assumed to not exist by definition. Shouldn't the components that require rendering be responsible for setting it up completely, including BRDF if needed?

On the other side, it could be useful and a good quality of life improvement if the bootstrap component would provide a minimal required pipeline (i.e. BRDF). Alternatively, I was thinking about a bootstrap bus that would handle BRDF creation, and then use it here instead of all this code.

What do you think? (cc @michalpelka)

Reply:
@moudgils @pijaro I've prepared fix in development O3DE:
https://github.com/o3de/o3de/pull/18531